InnerEye/ML/utils/io_util.py

```diff
@@ -209,6 +209,19 @@ def load_dicom_image(path: PathOrString) -> np.ndarray:
     reader.SetFileName(str(path))
     image = reader.Execute()
     pixels = sitk.GetArrayFromImage(image)
+
+    reader.ReadImageInformation()
+    if reader.GetMetaData("0028|0004").strip() == "MONOCHROME1":  # DICOM TAG "PhotometricInterpretation"
+        # invert image so bit interpretation is like MONOCHROME2, where a 0 bit is black
+        bits_stored = int(reader.GetMetaData("0028|0101"))  # DICOM TAG "BitsStored"
+        pixel_repr = int(reader.GetMetaData("0028|0103"))  # DICOM TAG "PixelRepresentation"
```

Comment:
You are using string literals here, that you repeat later in your get_mock_funciton in the tests. That has the risk of running out of sync, move the literals into constants.